src/components/Sum.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a, b }) => (
+  <p className="Numbers">
+    {`Sum of ${a} and ${b} is ${a + b}`}
+  </p>
+);
+
+Sum.defaultProps = {
+  b: 0,
+};
+
+Sum.propTypes = {
+  a: PropTypes.number.isRequired,
+  b: PropTypes.number,
+};
```

Comment:
Don't clearly understand why `a` is required and `b` is not. But OK.

Reply:
> Don't clearly understand why `a` is required and `b` is not. But OK.

Such a decision didn't have any sense, just practiced. For self-affirmation)